app/experimenter/experiments/models.py

```diff
@@ -613,8 +613,17 @@ def allocate_bucket_range(self):
     @property
     def can_edit(self):
         return (
-            self.status == self.Status.DRAFT
-            and self.publish_status == self.PublishStatus.IDLE
+            (
+                self.status == self.Status.DRAFT
+                and self.publish_status == self.PublishStatus.IDLE
+            )
+            or (
+                self.status == self.Status.LIVE
+                and (
+                    self.publish_status == self.PublishStatus.DIRTY
+                    or self.publish_status == self.PublishStatus.IDLE
+                )
+            )
```

Comment:
Shouldn't this be guarded on `is_rollout`?

Reply:
good point 👍

Reply:
I split out the tests so there's one for experiments and one for rollouts. The parameterization would have been too huge otherwise